src/pages/ProjectStart/Components/CardItem.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+
+function CardItem(props) {
+  const { title, redTitle, buttonText } = props;
```

Comment:
```suggestion
function CardItem({ title, redTitle, buttonText }) {
```
함수 파라미터 자리에서 직접 구조분해 할당 가능합니다!

Reply:
오옹..역시 구조분해를 사랑하시는 종택님